docusaurus/docs/test-a-plugin/getting-started/write-tests.md

```diff
@@ -0,0 +1,96 @@
+---
+id: writing-tests
+title: Writing tests
+description: How to write tests for Grafana E2E
+draft: true
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - e2e
+  - example test
+sidebar_position: 2
+---
+
+# Write tests
+
+After you have [set up Grafana Plugin E2E](installation.md), you are ready to write your first test. This guide provides sample tests that you can use to test your plugin against multiple versions of Grafana at once.
```

Comment:
```suggestion
After you have [set up Grafana Plugin E2E](installation.md), you are ready to write your first test. 
```

The part about "multiple versions of Grafana at once" may be a bit misleading in this context. Think we can remove this sentence.